docs/guide.rst

```diff
@@ -98,20 +98,32 @@ Here's an example how to use it.
     Signing Declare transactions is possible only with Accounts having `__validate__` entrypoint (with `supported_tx_version = 1`).
 
 
-Deploying new contracts
------------------------
+Deploying contracts
+-------------------
 
-Here's how you can deploy new contracts:
+Simple declare and deploy
+#########################
 
-.. codesnippet:: ../starknet_py/tests/e2e/docs/guide/test_deploying_new_contracts.py
+The simplest way of declaring and deploying contracts on the StarkNet is to use the :ref:`Contract` class.
```

Comment:
Add info informing what is done under the hood (first Declare tx, then Invoke tx to the UDC).